builder/parser/line_parsers.go

```diff
@@ -149,7 +149,7 @@ func parseEnv(rest string) (*Node, map[string]bool, error) {
 		strs := TOKEN_WHITESPACE.Split(rest, 2)
 
 		if len(strs) < 2 {
-			return nil, nil, fmt.Errorf("ENV must have two arguments")
+			return nil, nil, fmt.Errorf(key + " must have two arguments")
```

Comment:
Actually, discussing with @vieux, it's true that isn't so much of an issue. In the end, it's like saying that the command-line has a "syntax sugar" that defaults `--label plop` to `--label plop=""`. The important thing is to allow filtering on the existence of a given key, rather than a specific value, and this is independent.